### PR TITLE
Astro 3/14 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Relies on https://github.com/astronomer/docs/pull/2006 but I can also remove the link if we want to just get these out

Based on https://app.zenhub.com/workspaces/team-infra-61113e60aad91d000feb81bf/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODk1Mzk

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- Formerly located in the **Settings** tab, general Organization settings are now in **Settings** > **General**.
+
+### New Astro Cloud IDE integration with GitLab
+
+You can now configure a GitLab repository in your Astro Cloud IDE project. Configuring a GitLab repository allows you to commit your pipelines and deploy them to Astro directly from the Astro Cloud IDE. See [link].
+
+### Additional improvements
+
+- Clusters on an Astro - Hosted installation no longer retain Airflow logs which are older than 90 days.
+- The Airflow node pool instance type on GCP clusters has been reduced from `n2-standard-4` to `n2-standard-2`.
```
Comment (chomipi88) [inline astro/release-notes.md:45]:
```suggestion
- The Data Plane System node pool instance type on GCP clusters has been reduced from `n2-standard-4` to `n2-standard-2`.
```

File: astro/resource-reference-gcp.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Read the following document for a reference of our default resources as well as
...
-| Airflow node pool                                                                                    | A node pool that runs all core Airflow components, including the scheduler and webserver, for all Deployments in the cluster. This node pool is fully managed by Astronomer.                                                                                                              | 1x pool of n2-standard-4 nodes                                                             |                                                                                  |
+| Airflow node pool                                                                                    | A node pool that runs all core Airflow components, including the scheduler and webserver, for all Deployments in the cluster. This node pool is fully managed by Astronomer.                                                                                                              | 1x pool of n2-standard-2 nodes                                                             |                                                                                  |
```
Comment (chomipi88) [inline astro/resource-reference-gcp.md:19]: No change here

File: astro/resource-reference-gcp.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Read the following document for a reference of our default resources as well as
...
 | Astro system node pool                                                                               | A node pool that runs all other system components required in Astro. The availability zone determines how many nodes are created. This node pool is fully managed by Astronomer.                                                                                                          | 1x pool of n2-standard-2 nodes                                                             |                                                                                  |
```
Comment (chomipi88) [inline astro/resource-reference-gcp.md:20]: The change is here but looks like we already have the new node type documented.

File: astro/resource-reference-gcp.md (hunk 2 above)
Comment (jwitz) [inline astro/resource-reference-gcp.md:19]:
```suggestion
| Airflow node pool                                                                                    | A node pool that runs all core Airflow components, including the scheduler and webserver, for all Deployments in the cluster. This node pool is fully managed by Astronomer.                                                                                                              | 1x pool of n2-standard-4 nodes                                                             |                                                                                  |
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -23,6 +23,40 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+While most tabs were migrated directly to the left menu with the same name, some pages have been renamed and moved:
+
+- Formerly located in **Overview**, your Workspace list is now available in **Workspaces**.
+- Formerly located in the **People** tab, Organization user management settings are now in **Settings** > **Access Management**.
+- Formerly located in the **Settings** tab, general Organization settings are now in **Settings** > **General**.
+
+### New Astro Cloud IDE integration with GitLab
+
+You can now configure a GitLab repository in your Astro Cloud IDE project. Configuring a GitLab repository allows you to commit your pipelines and deploy them to Astro directly from the Astro Cloud IDE. See [link].
```
Comment (jwitz) [inline astro/release-notes.md:53]:
```suggestion
You can now configure a GitLab repository in your Astro Cloud IDE project. Configuring a GitLab repository allows you to commit your pipelines and deploy them to Astro directly from the Astro Cloud IDE. See [cloud-ide/deploy-project.md#deploy-a-project-from-a-git-repository-to-astro].
```

**Pull request merged**